src/Console/ConsoleIo.php

```diff
@@ -123,6 +128,15 @@ public function __construct(
         $this->_helpers->setIo($this);
     }
 
+    /**
+     * @param bool $value Value
+     * @return void
+     */
+    public function setInteractive(bool $value): void
```

Comment:
We've been trying to avoid adding public methods in bugfix releases. Should this move to 4.next?

Reply:
We can do that.